src/frontends/tensorflow_common/src/op/addN.cpp

```diff
@@ -18,15 +18,28 @@ OutputVector translate_add_n_op(const NodeContext& node) {
     default_op_checks(node, 1, {"AddN", "ADD_N"});
     int num_size = static_cast<int>(node.get_input_size());
 
-    Output<Node> result = node.get_input(0);
+
+    auto result = node.get_input(0);
     for (int ind = 1; ind < num_size; ++ind) {
         result = make_shared<v1::Add>(result, node.get_input(ind));
     }
```

Comment:
please think how you will add operands if they have `ComplexTypeMark` under. Is it valid?